mautwitdm/uploader.py

```diff
@@ -83,7 +83,7 @@ async def upload(self, data: bytes = None, url: str = None, mime_type: str = Non
             category = "dm_video"
             size_limit = 15 * 1024 * 1024
         else:
-            raise ValueError("Unsupported mime type")
+            raise NotImplementedError(f"Unsupported mime type {mime_type}")
         if len(data) > size_limit:
             raise ValueError("File too big")
```

Comment:
This should probably also send an unsupported checkpoint (limits are 5/15 mb, usual matrix limit is 100mb)